docs/source/other-learning-materials.rst

```diff
@@ -10,6 +10,11 @@ Learning materials
 
 Here are some slides and videos you may want to check out while getting started or when training other lab members!
 
+.. admonition:: "Children Helping Science" vs "Lookit"
+
+   Note that most of these materials were created prior to the merger between Children Helping Science and Lookit, and therefore use the "Lookit" term to refer to the entire site infrastructure. We now refer to the "Lookit" website/platform as "Children Helping Science" (CHS), and we still use the "Lookit" term for the internal experiment runner.
+
+
```

Comment:
@mekline another note about CHS vs Lookit terms on this "other learning materials" page, because most of these resources use "Lookit" to refer to the whole platform/project.